### PR TITLE
ProcMod: release envbus when finished

### PR DESCRIPTION
**Pull request by dyfer**
It seems that we were not releasing ProcMod's envbus when finished.
This fixes that, so the bus allocator is properly cleared (including re-triggering  procmods, I believe).

### CODE REVIEW

File: ProcMod.sc [hunk 1]
```diff
@@ -239,7 +242,10 @@ ProcMod {
 		currelfunc = releaseFunc;
 		curccctrl = ccCtrl;
 		isRunning.if({server.sendMsg(\n_free, curgroup)});
-		isReleasing.if({oldgroups.do({arg me; server.sendMsg(\n_free, me)})});
+		isReleasing.if({
+			oldgroups.do({arg me; server.sendMsg(\n_free, me)});
+			oldenvbuses.do({arg me; server.controlBusAllocator.free(me)});
```
Comment (joshpar) [inline ProcMod.sc:247]: These shouldn't be freed while it is releasing... This should happen in 'clear'

**Review by joshpar — CHANGES_REQUESTED** [review]
looks good! I think just one thing needs to be cleared up

File: ProcMod.sc (hunk 1 above)
Comment (dyfer) [reply ProcMod.sc:247]: I wasn't super clear on the logic here, but current group is added to `oldgroups` array when the proc *is releasing*, and the only place where `oldgroups` is being used is in `-kill`, to free resources of all instances of the process I assume.

The current envbus is still freed in [`-clear`](https://github.com/supercollider-quarks/JoshMisc/pull/5/files#diff-22f07476de3ee06686252b90b3528164R269). What do you think?

**Comment by dyfer** [comment]
@joshpar not sure if you've seen the question I had w/r/t your change request? I'd like to hear your opinion how to proceed. Thanks!

File: ProcMod.sc (hunk 1 above)
Comment (joshpar) [reply ProcMod.sc:247]: So - I just typed up a big response, and while doing so, I think I figured out where I was incorrect and you are correct :) - thanks for your patience on that!

**Review by joshpar — APPROVED** [review]

**Comment by dyfer** [comment]
Thanks Josh!

**Pull request merged**